arch/arm/src/sam34/sam_hsmci.c

```diff
@@ -1800,6 +1802,9 @@ static int sam_sendcmd(FAR struct sdio_dev_s *dev,
          cmd, arg, regval);
   putreg32(regval, SAM_HSMCI_CMDR);
   sam_cmdsample1(SAMPLENDX_AFTER_CMDR);
+
+  nxsig_usleep(10);
```

Comment:
Why do we need a sleep here?

Reply:
In my testing, without the delay, card initialisation was unsuccessful.
Card initialisation is successful when mcinfo is turned on. This is because of the added execution time to run mcinfo @ line 1801.
Inserting the delay replicates this behaviour.

It appears the timing between sam_sendcmd and sam_waitresponse is too short.

Happy to investigate better ways to achieve this if you wish?

Reply:
In general it is good to know what is happening in the system. If you have it working with this delay I do not want to block changes from merge. Maybe adding some comment in code similar you posted here could be useful if somebody will ht any issue in the future and will decide to investigate it.

Reply:
Great idea. I have added a comment around this delay.

Reply:
Thank you